Xamarin.Forms.Core/TypeTypeConverter.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Xamarin.Forms
 {
+	[Xaml.ProvideCompiled("Xamarin.Forms.Core.XamlC.TypeTypeConverter")]
```

Comment:
it references the class by name, so the Core doesn't hold a reference on the BuildTask